README.md

```diff
@@ -479,23 +479,25 @@ tests:
   - dbt_expectations.expect_column_values_to_be_increasing:
       sort_column: date_day
       row_condition: "id is not null" # (Optional)
-      strictly: true # (Optional for comparison operator. Default is 'true', and it uses '>'. If set to 'flase' it uses '>='.)
+      strictly: true # (Optional for comparison operator. Default is 'true', and it uses '>'. If set to 'false' it uses '>='.)
+      partition_columns: partition_col_1, partition_col_2 # (Optional: default is None)
```

Comment:
Let's make `partition_columns` an actual list not a string, similar to https://github.com/calogica/dbt-expectations#expect_table_row_count_to_be_between